src/t8_vtk/t8_vtk_parallel.hxx

```diff
@@ -32,9 +32,9 @@ along with t8code; if not, write to the Free Software Foundation, Inc.,
 
 /**
  * Given a filename to a parallel vtk file (for example .pvtu) and its data files, 
- * read a piece of the data files (like .vtu, .vtp, ...).
+ * read a piece of the data files (like .vtu, ...).
  * 
- * \param[in] filename  The name of a parallel vtk file (.pvtu for example)
+ * \param[in] filename  The name of a parallel vtk file to a distributed vtkUnstructuredGrid
  * \param[in] grid On input a vtkSmartPointer, that will hold the grid described
  *                 by the pieces read on this proc. 
  * \returns        non-zero on success, zero if the reading failed. 
```

Comment:
I recommend to be explicit: `t8_read_parallel_unstructured`.

Reply:
Done